docs/overview.rst

```diff
@@ -348,7 +349,4 @@ Cloud Console.
 .. _westeurope.azure.cratedb.cloud: https://westeurope.azure.cratedb.cloud/
 .. |Microsoft Azure| raw:: html
 
-    <a href="https://azure.microsoft.com/" target="_blank">Microsoft Azure</a>
-.. |region| raw:: html
-
-    <a href="https://azure.microsoft.com/en-us/global-infrastructure/regions/" target="_blank">region</a>
\ No newline at end of file
+    <a href="https://azure.microsoft.com/" target="_blank">Microsoft Azure</a>
```

Comment:
please shout (gently) at me if I'm wrong, but didn't we agree to remove these types of links? if not, or if I'm missing something, just ignore me (but pls tell me haha)

Reply:
This is still being handled as part of the website redesign, so I'm just leaving them as-is for now until it's resolved in some definitive fashion